### DISCUSSION TITLE
pnp-modern-search 4.10 vs 4.12

### DISCUSSION THREAD

**Discussion by waldek9** [microsoft-search/pnp-modern-search#3895] [Q&A] [answered]
Short question - if I have 4.10 already in my env and I want to add 4.12 - does it have any influence on existing webparts? I would like to avoid scenario when I install new solution and then suddently all existing 4.10 webparts are stopped...

**Comment by kasperbolarsen** [comment, marked as answer]
Nope, I have upgraded 20+ tenants, and haven't seen any issues. 
The only issue in 4.12.2 is that you must grant a  new delegated permission in order for new search verticals to work, as this web part now supports audience targeting
